providers/openshift/provider.go

```diff
@@ -287,12 +287,10 @@ func parseResources(resources string) (recordsByPath, error) {
 // Complete performs final setup on the provider or returns an error.
 func (p *OpenShiftProvider) Complete(data *providers.ProviderData, reviewURL *url.URL) error {
 	if emptyURL(reviewURL) {
-		if emptyURL(data.LoginURL) {
```

Comment:
We probably want to keep —Openshift-review-url working as an override, and if it’s not set then do the on demand lookups.

Reply:
If I understand the code right, this does not remove the override, it only removes the failure should we fail to fetch `LoginURL`.